web/styles/portico/billing.css

```diff
@@ -374,4 +374,12 @@ input[name="licenses"] {
         cursor: pointer;
         background-color: hsl(0deg 0% 100%);
     }
+
+    & input[name="website"] {
+        width: 100%;
+    }
+
+    & textarea[name="description"] {
+        width: 100%;
+    }
```

Comment:
Also for the record, tweaked this commit to use the syntax of 5cdf38b1f7286939e2128684ecef929ad0a8f797 for these selectors